docs/en/workbench/how_to/create_workbench.mdx

```diff
@@ -0,0 +1,60 @@
+---
+weight: 14
+---
+
+# Create Workbench
+
+## Prerequisites
+
+- Ensure you have `kubectl` configured and connected to your cluster.
+- Ensure you have created `WorkspaceKind` (cluster resource).
+
+### Steps
+
+<Steps>
+### Create PVC
+  1. Login, go to the **Alauda Container Platform** page.
+  2. Click **Storage** > **PersistentVolumeClaims** to enter the **PVC** list page.
+  3. Find the **Create PVC** button, click **Create**, and enter the info.
+
+### Create SA
+  `kubectl` apply the example yaml as blow.
+
+  ```yaml
+  apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: workbench-editor
+    namespace: NAMESPACE_NAME # [!code callout]
+  ---
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: workbench-sa-binding
+  subjects:
+  - kind: ServiceAccount
+    name: workbench-editor
+    namespace: NAMESPACE_NAME # [!code callout]
+  roleRef:
+    kind: ClusterRole
+    name: aml-namespace-editor
+    apiGroup: rbac.authorization.k8s.io
+  ```
+  <Callouts>
+    1. NAMESPACE_NAME should be same with the namespace of workbench created.
+    2. Same as above.
+  </Callouts>
+
+</Steps>
+
+## Create Workbench by using the web console
+
+### Procedure
+
+1. Login, go to the **Alauda AI** page.
+2. Click **Workbench** to enter the **Workbench** list page.
+3. Find the **Create** button, click **Create**, you will enter the creation form, and you can create a workbench after filling in the information.
+
+## Connect to Workbench
+
+After creation is complete, click `Workbench` in the left navigation to enter the workbench list page. Each data entry represents an instance. After the status is `Running`, click the Connect button to open the corresponding instance page.
```

Comment:
After creating a workbench instance, click `Workbench` in the left navigation bar, you workbench instance should show up here in the list. When the status becomes `Running` click the `Connect` button to enter the workbench, then you are ready to go.